frontend/src/components/LessonsPage/LessonPage.js

```diff
@@ -59,47 +64,39 @@ const LessonPage = (props) => {
 
     useEffect(() => {
         if (groupId) {
-            getLessonsByGroup(groupId);
+            getLessonsByGroupStart(groupId);
         }
     }, [groupId]);
 
     useEffect(() => {
-        showAllTeachersService();
-        getLessonTypes();
-        showAllGroupsService();
+        showAllTeachers();
+        getLessonTypesStart();
+        getEnabledGroupsStart();
         showAllSubjectsService();
         showAllSemestersService();
```

Comment:
@Yashchemskyi-Oleksandr we have saga for showAllSemestersService()

Reply:
done